docs/source/orchestration.rst

```diff
@@ -23,6 +25,29 @@ To perform integration with Azure Data Factory, please do the following steps:
         --resource-group some-group \
         --factory-name some-factory \
         --name some-pipeline-name
+        
+2. Deploy from CD pipeline
+
+Inside your CD pipeline, add Azure login before the ``dbx datafactory reflect`` step. Below is an example using GitAction:
```

Comment:
It's GitHub Actions I assume :) 

Also, CI and CD pipelines might have different meanings in different contexts. Please use the wider term - CI/CD pipeline. 

Also, the name of the section is a bit incorrect, since you'll need to do three steps:
- Deploy the job files (dbx deploy --files only)
- Login with AZ credentials
- Reflect job definitions to Azure

Reply:
please see the updated commit [here](https://github.com/yinxi-db/dbx/blob/patch-2/docs/source/orchestration.rst)